packages/util/src/bytes.ts

```diff
@@ -276,3 +276,15 @@ export const baToJSON = function (ba: any): any {
     return array
   }
 }
+
+/**
+ * Checks provided Buffers for leading zeroes and throws if found.
```

Comment:
It would be good if we add some example values here in the code docs, something like:

Valid: `0x1`, `0x`, `0x01` (again, see question on on the `0x1`, `0x01` case)
Invalid: `0x0`, `0x00`, `0x0001`

(not 100% sure I got all, also there might be more expressive example values out there 🙂)